pkg/dlx/handler.go

```diff
@@ -101,14 +109,39 @@ func (h *Handler) handleRequest(res http.ResponseWriter, req *http.Request) {
 		res.WriteHeader(http.StatusInternalServerError)
 		return
 	}
+	h.logger.DebugWith("success creating targetURL")
+
+	//if in cache, do not log
```

Comment:
```suggestion
	// if in cache, do not log
```

instead of comment what, comment why